backends/source/dppl_sycl_queue_interface.cpp

```diff
@@ -103,9 +103,24 @@ void error_reporter (const std::string & msg)
 class QMgrHelper
 {
 public:
-    static std::vector<cl::sycl::queue>   cpu_queues;
-    static std::vector<cl::sycl::queue>   gpu_queues;
-    static thread_local std::vector<cl::sycl::queue> active_queues;
+    static std::vector<cl::sycl::queue>&   cpu_queues_()
```

Comment:
Minor nitpick.  Can you please add a space before the open paren? I usually ffollow the convention of adding a space before the open paren in function declaration. In the long term let us create a common coding style and formatter for VS Code so that everyone can be consistent.